res/lang/gen_settings/genSettings_ua.txt

```diff
@@ -0,0 +1,202 @@
+WEEK_SHORT_SUNDAY    = Нд
+WEEK_LONG_MONDAY     = Понеділок
+WEEK_LONG_TUESDAY    = Вівторок
+WEEK_LONG_WEDNESDAY  = Середа
+WEEK_LONG_THURSDAY   = Четвер
+WEEK_LONG_FRIDAY     = П’ятниця
+WEEK_LONG_SATURDAY   = Субота
+WEEK_LONG_SUNDAY     = Неділя
+SEASON_SPRING        = весна
+SEASON_SUMMER        = літо
+SEASON_AUTUMN        = осінь
+SEASON_WINTER        = зима
+
+
+MENU_SOLO_GAME       = Одиночна гра
+MENU_START_GAME      = Почати гру
+MENU_NETWORKGAME     = Гра по локальній мережі
+MENU_ONLINEGAME      = Онлайн-гра
+MENU_JOIN            = Приєднатися
+MENU_CREATE_GAME     = Створити гру
+MENU_BACK            = Назад
+MENU_SETTINGS        = Налаштування
+MENU_QUIT            = Вийти з гри
+MENU_AVAILABLE_GAMES = Доступні ігри
+
+ABBREVIATION_THOUSAND = Т
+ABBREVIATION_MILLION  = М
+ABBREVIATION_BILLION  = Млрд
+
+DIFFICULTY_EASY   = Легко
+DIFFICULTY_NORMAL = Нормально
+DIFFICULTY_HARD   = Важко
+DIFFICULTY_CUSTOM = Налаштовано
+
+GAMETIME  = Час
+PLAYER    = Гравець
+GAME      = Гра
+GAME_TIME = Ігровий час
+GAMEDAY   = Ігровий день
+
+
+GAMESPEED      = Швидкість гри
+SET_SPEED_TO_X = Встановити швидкість гри на %SPEED%
+
+MANUAL    = Посібник
+HELP      = Допомога / Пауза
+SHOW_HELP = Показати допомогу
+
+DO_NOT_SHOW_AGAIN        = Більше не показувати
+DO_NOT_SHOW_ANY_TIPS     = Не показувати підказки
+SHOW_INTRODUCTORY_GUIDES = Показувати вступні поради
+
+ERROR_NOT_POSSIBLE                  = Неможливо!
+ERROR_NOT_ABLE_TO_SELL_LAST_STATION = Неможливо продати останню станцію.
+ERROR_NOT_ENOUGH_MONEY              = Недостатньо грошей!
+ERROR_NOT_ENOUGH_MONEY_TEXT         = У вас недостатньо грошей для цього.
+
+NEWGAMESETTINGS_RANDOMIZE          = Випадкове значення
+NEWGAMESETTINGS_RANDOMIZE_DETAIL   = Випадковий вигляд (колір, фігура) та імена
+NEWGAMESETTINGS_CHANNELNAME_DETAIL = Назва телевізійного каналу гравця
+NEWGAMESETTINGS_PLAYERNAME_DETAIL  = Ім’я гравця
+NEWGAMESETTINGS_DIFFICULTY         = Складність
+NEWGAMESETTINGS_DIFFICULTY_DETAIL  = Рівень складності для цього гравця
+NEWGAMESETTINGS_START_YEAR_DETAIL  = Рік початку гри. Впливає на доступні програми!
+NEWGAMESETTINGS_GAME_SEED          = Номер гри / seed
+NEWGAMESETTINGS_GAME_SEED_DETAIL   = Однакові номери гри ведуть до подібних стартових умов (пропозицій від продавців).
+
+MESSAGE_CLOSES_AT_TIME = Повідомлення закриється о %TIME%
+MESSAGE_CLOSES_AT_DAY  = Повідомлення закриється %TIME%
+
+
+HINT_PRESSING_ALT_WILL_SHOW_DETAILS    = Натискання ALT покаже деталі.
+HINT_PRESSING_ALT_WILL_FIX_TOOLTIP     = Натискання ALT закріпить підказку.
+HINT_PRESSING_ALT_WILL_RELEASE_TOOLTIP = Натискання ALT відпустить підказку.
+HINT_PROGRAMMEPLANER_SHORTCUTS         = ALT: перемикання таблиці | CTRL+Click: копіювати програму | SHIFT+Click: наступний епізод
+HINT                                   = Підказка
+
+STARTHINT_TITLE1 = Планування програм
+STARTHINT_TEXT1  = CTRL дозволяє розміщувати програму кілька разів у плані, а SHIFT розміщує наступний епізод серії.
```

Comment:
CTRL SHIFT...

Reply:
the same, it is not translatable, they also appear as CTRL, SHIFT, ALT on localized keyboard